IFC 4.3/Georeferencing_Tiny_Survey_1/UT_one_string_one_pont.ifc

```diff
@@ -0,0 +1,74 @@
+#11 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#12 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#13 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#14 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #16, #20);
+#15 = IFCLOCALPLACEMENT($, #16);
+#16 = IFCAXIS2PLACEMENT3D(#17, #18, #19);
+#17 = IFCCARTESIANPOINT((0., 0., 0.));
+#18 = IFCDIRECTION((0., 0., 1.));
+#19 = IFCDIRECTION((1., 0., 0.));
+#20 = IFCDIRECTION((0., 1., 0.));
+#21 = IFCPROJECTEDCRS('EPSG:28356', 'MGA94 Zone 56 and AHD', $, 'EPSG:5711', $, $, $);
+#22 = IFCMAPCONVERSION(#14, #21, 0., 0., 0., 1., 0., 1.);
+#23 = IFCSITE('2w45iIdVr8fBWGawWXW1N1', #2, 'Default Site', 'Description of Default Site', $, #15, $, $, .ELEMENT., $, $, $, $, $);
+#24 = IFCRELAGGREGATES('0zkNxXbbn85QpmKGTxzn6G', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#23));
+#25 = IFCBUILDING('2OYQzmOTP12A5bwbGOhUP0', #2, 'one string and one point', 'Description of one string and one point', $, #15, $, $, .ELEMENT., $, $, $);
+#26 = IFCRELAGGREGATES('2UlhmNw$b6YR3qyaSYovjk', #2, 'SiteContainer', 'SiteContainer For All Models', #23, (#25));
+#27 = IFCRELCONTAINEDINSPATIALSTRUCTURE('1wG9_o$RjEB9yHwLmbJVqI', #2, 'Building Container', 'BuildingContainer for Elements', (#28, #53), #25);
+#28 = IFCBUILDINGELEMENT('0tm1T3p5fEjRgwNfA4AXSw', #2, 'RD ET', 'test', 'Super', #15, #29, $, $);
+#29 = IFCPRODUCTDEFINITIONSHAPE($, $, (#30));
+#30 = IFCSHAPEREPRESENTATION(#14, 'Body', 'GeometricCurveSet', (#31));
+#31 = IFCGEOMETRICCURVESET((#32));
+#32 = IFCPOLYLINE((#33, #34, #35, #36, #37, #38, #39, #40, #41, #42, #43, #44, #45, #46, #47));
+#33 = IFCCARTESIANPOINT((256561.639011591, 7011556.07984224, 39.2638916532321));
+#34 = IFCCARTESIANPOINT((256543.68, 7011579.17, 40.21));
+#35 = IFCCARTESIANPOINT((256528.57, 7011599.07, 40.79));
+#36 = IFCCARTESIANPOINT((256503.87, 7011630.28, 41.74));
+#37 = IFCCARTESIANPOINT((256481.75, 7011657.75, 42.49));
+#38 = IFCCARTESIANPOINT((256465.35, 7011676.75, 43.48));
+#39 = IFCCARTESIANPOINT((256454.73, 7011687.96, 44.04));
+#40 = IFCCARTESIANPOINT((256444.35, 7011696.18, 44.37));
+#41 = IFCCARTESIANPOINT((256432.7, 7011704.32, 44.77));
+#42 = IFCCARTESIANPOINT((256421.91, 7011710.57, 45.34));
+#43 = IFCCARTESIANPOINT((256410.16, 7011716.75, 45.58));
+#44 = IFCCARTESIANPOINT((256395.03, 7011722.26, 45.83));
+#45 = IFCCARTESIANPOINT((256381.65, 7011725.57, 45.91));
+#46 = IFCCARTESIANPOINT((256372.3, 7011727.61, 45.91));
+#47 = IFCCARTESIANPOINT((256368.305104578, 7011728.47766959, 45.91));
+#48 = IFCSURFACESTYLE($, .POSITIVE., (#49));
+#49 = IFCSURFACESTYLERENDERING(#50, $, $, $, $, $, IFCNORMALISEDRATIOMEASURE(3.90625E-3), IFCSPECULAREXPONENT(10.), .NOTDEFINED.);
+#50 = IFCCOLOURRGB('grey', 9.01960784313726E-1, 9.01960784313726E-1, 9.01960784313726E-1);
+#51 = IFCPRESENTATIONSTYLEASSIGNMENT((#48));
+#52 = IFCSTYLEDITEM(#31, (#51), $);
+#53 = IFCBUILDINGELEMENT('18x1jaUq54pBDn0OQObE1f', #2, 'SF LE', 'test', 'Super', #15, #54, $, $);
```

Comment:
It's `IfcBuiltElement` [documentation](https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC1/HTML/schema/ifcproductextension/lexical/ifcbuiltelement.htm)

Reply:
resolved in b97e6fa